### PR TITLE
Feat/add docker test for wg

### PR DESCRIPTION
**Pull request by VendettaReborn**
### 🤔 This is a ...

- [ ] New feature
- [ ] Bug fix
- [ ] Performance optimization
- [x] Enhancement feature
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Workflow
- [ ] Other (about what?)

### 💡 Background and solution

Motivation: I wish to change some code in wireguard module, but the test is hard, i can only test it by running the whole app

### 📝 Changelog

feat: add docker test for wg

### ☑️ Self-Check before Merge

⚠️ Please check all items below before requesting a reviewing. ⚠️

- [ ] Doc is updated/provided or not needed
- [ ] Changelog is provided or not needed

### CODE REVIEW

File: clash_lib/src/proxy/wg/wireguard.rs [hunk 1]
```diff
@@ -69,6 +69,8 @@ impl WireguardTunnel {
         packet_writer: Sender<(PortProtocol, Bytes)>,
         packet_reader: Receiver<Bytes>,
     ) -> Result<Self, Error> {
+        tracing::debug!("config: private key: {:x?}, endpoint public key: {:x?}, remote endpoint: {}, source peer ip: {}, source peer ipv6: {:?}, keepalive seconds: {:?}, allowed ips: {:?}, reserved bits: {:?}",
+            config.private_key.to_bytes(), config.endpoint_public_key.to_bytes(), config.remote_endpoint, config.source_peer_ip, config.source_peer_ipv6, config.keepalive_seconds, config.allowed_ips, config.reserved_bits);
```
Comment (ibigbug) [inline clash_lib/src/proxy/wg/wireguard.rs]:
```suggestion
        tracing::debug!("config: endpoint public key: {:x?}, remote endpoint: {}, source peer ip: {}, source peer ipv6: {:?}, keepalive seconds: {:?}, allowed ips: {:?}, reserved bits: {:?}",
            config.endpoint_public_key.to_bytes(), config.remote_endpoint, config.source_peer_ip, config.source_peer_ipv6, config.keepalive_seconds, config.allowed_ips, config.reserved_bits);
```

Comment (ibigbug) [reply clash_lib/src/proxy/wg/wireguard.rs]: please don't print out the private key in logs

Comment (VendettaReborn) [reply clash_lib/src/proxy/wg/wireguard.rs]: sorry, it's a line for debug, forgot to delete

**Review by ibigbug — APPROVED** [review]
lgtm

**Comment by VendettaReborn** [comment]
> lgtm

great

**Pull request merged**